sites/www.thunderbird.net/thunderbird/all/index.html

```diff
@@ -24,11 +24,19 @@ <h2 class="split-text" aria-label="{{ _('Thunderbird Desktop.') }}">
           <p>{{ _('These are officially maintained and distributed by Thunderbird. Download the latest Release version unless you wish to test the Beta version and report bugs.') }}</p>
         </div>
         <div class="section-text">
-          <div id="daily-warning" class="warning-panel hidden">
-            <p>{{ _('The Daily version is an unstable testing and development platform, make sure you back up important data regularly!') }}</p>
-          </div>
-          <div id="release-warning" class="warning-panel hidden">
-            <p>{{ _('Thunderbird Release is available for testing purposes only until releases are deemed stable enough for official support. Make sure you backup important data regularly!') }}</p>
+          <div class="alerts" role="alert">
```

Comment:
I can't get role="alert" to properly work on voiceover with firefox. It sometimes works in safari. So this is a little spotty but it _should_ work.